TeamCode/src/main/java/org/firstinspires/ftc/teamcode/BestSTATEAutoEver.java

```diff
@@ -0,0 +1,1590 @@
+            sleep(500);
+            claw.setPosition(0);
+            sleep(500);
+            clarm.setTargetPosition(450);
+            clarm.setPower(-0.6);
+            sleep(500);
+        }
+
+
+
+/*
+        GoTov2(0,24, 0.25, 0, 6,6,true,false,0.012,true);
+        clarm.setTargetPosition(500);
+        clarm.setPower(0.5);
+        GoTov2(0,24, 0.25, 0, 6,6,false,false,0.012,true);
+        clarm.setTargetPosition(0);
+        clarm.setPower(-0.5);
+        GoTov2(0,24, 0.25, 0, 6,6,false,true,0.012,true);
+        GoTov2(0,-24, 0.25, 0, 6,6,true,false,0.012,true);
+        clarm.setTargetPosition(500);
+        clarm.setPower(0.5);
+        GoTov2(0,-24, 0.25, 0, 6,6,false,false,0.012,true);
+        clarm.setTargetPosition(0);
+        clarm.setPower(-0.5);
+        GoTov2(0,-24, 0.25, 0, 6,6,false,true,0.012,true);
+        sleep(1000);
+*/
+
+
+  /*wallTargetTracking(vuforia, allTrackables, 90, 0, 58, 0, 10, 2, 1, 3, false, 0);
+ /*
+        AngularAdjustment(-6.1, 0.02);
+
+        AngularAdjustment(-3.5, 0.028);
+
+        AngularAdjustment(1.15, 0.028);
+
+        GoTo(0, 12,.2, 0, 14, 0, true, true, 0.012, true);
+        if (QuadRun == 1) {
+            intakeFunction(16, 1300, 0, true, .03);
+            intakeFunction(6, 1300, 0, true, .03);
+        }
+/*
+        GoTo(0, 10,.2, 0, 14, 0, true, false, 0.012, true);
+        clarm.setTargetPosition(650);
+        clarm.setPower(0.3);
+        GoTo(10,10,0.2,0,14,0, false,true,0.012,true);
+        GoTo(-10,-10,0.2,0,14,0, true,false,0.012,true);
+        clarm.setTargetPosition(0);
+        clarm.setPower(-0.3);
+        GoTo(0, -10,.2, 0, 14, 0, false, true, 0.012, true);
+   /* GoTo(20, 20,.2, 0, 14, 0, true, true, 0.012, true);
+        GoTo(-20, -20,.2, 0, 14, 0, true, true, 0.012, true);
+        GoTo(-20, 20,.2, 0, 14, 0, true, true, 0.012, true);
+        GoTo(20, -20,.2, 0, 14, 0, true, true, 0.012, true);
+        GoTo(-20, -20,.2, 0, 14, 0, true, true, 0.012, true);
+        GoTo(20, 20,.2, 0, 14, 0, true, true, 0.012, true);
+        GoTo(20, -20,.2, 0, 14, 0, true, true, 0.012, true);
+        GoTo(-20, 20,.2, 0, 14, 0, true, true, 0.012, true);
+*/
+
+
+
+
+    /*
+    GoTo(0, 40, .3, 0, 14, 0, false, true, 0.012, true);
+IMUTurn(175);
+*/
+
+    /*
+    GoTo(20, -20, .3, 0, 14, 0, true, true, 0.012, true);
+        GoTo(20, 20, .3, 0, 14, 0, true, true, 0.012,true);
+        GoTo(0, -20, .3, 0, 14, 0, true, true, 0.012, true);
+        GoTo(-20, 20, .3, 0, 14, 0, true, true, 0.012, true);
+        GoTo(-20, -20, .3, 0, 14, 0, true, true, 0.012, true);
+        GoTo(0, 20, .3, 0, 14, 0, true, true, 0.012,true);
+
+*/
+
+        /*while(Xposition <= 24 || Yposition <= 48){
+    if (Xposition >= 24){
+        X = 0;
+    }
+    if (Yposition >= 48){
+        Y = 0;
+    }
+    MecanumFunction(0.1414 * Y,0.1 * X,0);
+    DistanceTracker();
+
+}
+
+         */
+        MecanumFunction(0, 0, 0);
+    }
+
+    private void DistanceTracker() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.